iOS/Services/StitchFCMService/StitchFCMService/StitchFCMService.h

```diff
@@ -0,0 +1,11 @@
+#import <UIKit/UIKit.h>
+
+//! Project version number for StitchFCMService.
+FOUNDATION_EXPORT double StitchFCMServiceVersionNumber;
+
+//! Project version string for StitchFCMService.
+FOUNDATION_EXPORT const unsigned char StitchFCMServiceVersionString[];
+
+// In this header, you should import all the public headers of your framework using statements like #import <StitchFCMService/PublicHeader.h>
```

Comment:
[nit] can remove some of these comments and extra lines

Reply:
:+1: